doc/Technical Documentation/Compiler Intermediate Language format.md

```diff
@@ -11,124 +11,126 @@ languages it is more similar to assembly languages than high level languages.
 
 As example, consider this extended hello world program:
 
-> \#include <stdio.h>
-> 
-> int main(int argc, char \*\*argv)
-> {
->   for (int i=0; i < 10; i++)
->       printf("hi %d\n", i);
-> }
-
+```c
+#include <stdio.h>
+ 
+int main(int argc, char **argv)
+{
+  for (int i=0; i < 10; i++)
+      printf("hi %d\n", i);
+}
+```
 when the /Y switch is used to compile it, the compiler dumps out the initial intermediate code representation as
 generated from parsing, and the optimizer dumps out the final intermediate code representation after all optimizations
 have been applied and registers have been assigned
 
 The initial representation looks something like this:
+```
```

Comment:
ASM is close, but I couldn't think of anything that would grab exactly what we wanted for syntax highlighting here so I decided to let it go because of the effort involved in trying to figure out custom Markdown Syntax Highlighting on github.